src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        final int arrayLenght = 6;
```

Comment:
```suggestion
        final int arrayLength = 6;
```
better make it a class constant